pages/handla.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="home">
+    <ProductMatrix />
+  </div>
+</template>
+
+<script>
+import ProductMatrix from '@/components/ProductMatrix.vue'
```

Comment:
Would now be a good time to rename this component? I recall you wanting to name it something else

Reply:
Not part of this pull request